citadel/indico_citadel/schemas_test.py

```diff
@@ -114,12 +111,12 @@ def test_dump_contribution(db, dummy_user, dummy_event, dummy_contribution, crea
 
 
 @pytest.mark.parametrize('scheduled', (False, True))
-def test_dump_subcontribution(db, dummy_user, dummy_event, dummy_contribution, create_entry, scheduled):
+def test_dump_subcontribution(db, dummy_user, dummy_event, dummy_contribution, create_timetable_entry, scheduled):
     from .schemas import SubContributionRecordSchema
 
     extra = {}
     if scheduled:
-        create_entry(dummy_contribution, utc.localize(datetime(2020, 4, 20, 4, 20)))
+        create_timetable_entry(cummy_event, dummy_contribution, utc.localize(datetime(2020, 4, 20, 4, 20)))
```

Comment:
```suggestion
        create_timetable_entry(dummy_event, dummy_contribution, utc.localize(datetime(2020, 4, 20, 4, 20)))
```

(funny typo though 😂)

Reply:
Definitely, especially in my language :D